src/main/kotlin/com/coder/gateway/cli/CoderCLIManager.kt

```diff
@@ -209,7 +209,7 @@ class CoderCLIManager(
      *
      * This can take supported features for testing purposes only.
      */
-    fun configSsh(workspaceNames: List<String>, feats: Features = features) {
+    fun configSsh(workspaceNames: Set<String>, feats: Features = features) {
```

Comment:
I changed this to a `set` since we are seeing duplicate entries in our ssh configs.

Reply:
We have :
```
resource "coder_agent_instance" "main" {
...
}
resource "coder_agent" "main" {
...
}
```
and were getting multiple entries with "main", so maybe related to `coder_agent_instance` colliding with `coder_agent`?

Reply:
Ahhhhhhhh that makes sense!  Thank you for the sleuthing.